### PR TITLE
Feature/hive table repair

### PR DESCRIPTION
**Pull request by jozefbakus**
Helps user to repair Hive table via spark

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
+    Copyright 2018 ABSA Group Limited
```
Comment (kevinwallimann) [inline README.md]: Should be 2023

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright 2021 ABSA Group Limited
```
Comment (kevinwallimann) [inline pom.xml]: 2023

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    <version>0.0.1-SNAPSHOT</version>
+    <description>Spark hive jobs</description>
+    <url>https://github.com/AbsaOSS/hyperdrive-hive-jobs</url>
+    <packaging>jar</packaging>
+
+    <properties>
+        <encoding>UTF-8</encoding>
+        <scala.version>2.11.12</scala.version>
+        <scala.compat.version>2.11</scala.compat.version>
+        <spark.version>2.4.3</spark.version>
```
Comment (kevinwallimann) [inline pom.xml]: How about version 2.4.7? Latest patch version

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        <maven.shade.plugin.version>3.2.1</maven.shade.plugin.version>
+        <commons.configuration2.version>2.7</commons.configuration2.version>
```
Comment (kevinwallimann) [inline pom.xml]: Can you try it out with a more recent version? I think 2.7 will immediately give us security warnings. You can do it as a separate issue.

File: scalastyle-config.xml [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<scalastyle commentFilter="enabled">
+  <name>Scalastyle standard configuration</name>
+  <check enabled="true" class="org.scalastyle.file.HeaderMatchesChecker" level="error">
+    <parameters>
+      <parameter name="header">/*
+ * Copyright 2018 ABSA Group Limited
```
Comment (kevinwallimann) [inline scalastyle-config.xml]: 2023

File: pom.xml (hunk 4 above)
Comment (jozefbakus) [reply pom.xml]: Done

File: pom.xml (hunk 3 above)
Comment (jozefbakus) [reply pom.xml]: Done

File: pom.xml (hunk 2 above)
Comment (jozefbakus) [reply pom.xml]: Done

File: README.md (hunk 1 above)
Comment (jozefbakus) [reply README.md]: Done

File: scalastyle-config.xml (hunk 5 above)
Comment (jozefbakus) [reply scalastyle-config.xml]: Done

**Review by kevinwallimann — APPROVED** [review]

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -0,0 +1 @@
+* @kevinwallimann @jozefbakus
```
Comment (kevinwallimann) [inline .github/CODEOWNERS:1]: Please add filiphornak as code owner

**Pull request merged**